apps/blockchain/scripts/generate_state_tests.exs

```diff
@@ -0,0 +1,2571 @@
+      "static_callcodecallcallcode_101_OOGMAfter",
+      "static_callcodecallcallcode_101_OOGMAfter2",
+      "static_callcodecallcallcode_101_OOGMAfter_1",
+      "static_callcodecallcallcode_101_OOGMAfter_3",
+      "static_callcodecallcallcode_101_OOGMBefore",
+      "static_callcodecallcallcode_101_OOGMBefore2",
+      "static_callcodecallcallcode_101_SuicideEnd",
+      "static_callcodecallcallcode_101_SuicideEnd2",
+      "static_callcodecallcallcode_101_SuicideMiddle",
+      "static_callcodecallcallcode_101_SuicideMiddle2",
+      "static_callcodecallcallcode_ABCB_RECURSIVE",
+      "static_callcodecallcallcode_ABCB_RECURSIVE2",
+      "static_callcodecallcodecall_110",
+      "static_callcodecallcodecall_1102",
+      "static_callcodecallcodecall_110_2",
+      "static_callcodecallcodecall_110_OOGE",
+      "static_callcodecallcodecall_110_OOGE2",
+      "static_callcodecallcodecall_110_OOGMAfter",
+      "static_callcodecallcodecall_110_OOGMAfter2",
+      "static_callcodecallcodecall_110_OOGMAfter_2",
+      "static_callcodecallcodecall_110_OOGMAfter_3",
+      "static_callcodecallcodecall_110_OOGMBefore",
+      "static_callcodecallcodecall_110_OOGMBefore2",
+      "static_callcodecallcodecall_110_SuicideEnd",
+      "static_callcodecallcodecall_110_SuicideEnd2",
+      "static_callcodecallcodecall_110_SuicideMiddle",
+      "static_callcodecallcodecall_110_SuicideMiddle2",
+      "static_callcodecallcodecall_ABCB_RECURSIVE",
+      "static_callcodecallcodecall_ABCB_RECURSIVE2",
+      "static_callcodecallcodecallcode_111_SuicideEnd",
+      "static_calldelcode_01",
+      "static_calldelcode_01_OOGE",
+      "static_contractCreationMakeCallThatAskMoreGasThenTransactionProvided",
+      "static_contractCreationOOGdontLeaveEmptyContractViaTransaction",
+      "static_log0_emptyMem",
+      "static_log0_logMemStartTooHigh",
+      "static_log0_logMemsizeTooHigh",
+      "static_log0_logMemsizeZero",
+      "static_log0_nonEmptyMem",
+      "static_log0_nonEmptyMem_logMemSize1",
+      "static_log0_nonEmptyMem_logMemSize1_logMemStart31",
+      "static_log1_MaxTopic",
+      "static_log1_emptyMem",
+      "static_log1_logMemStartTooHigh",
+      "static_log1_logMemsizeTooHigh",
+      "static_log1_logMemsizeZero",
+      "static_log_Caller",
+      "static_makeMoney",
+      "static_refund_CallA",
+      "static_refund_CallToSuicideNoStorage",
+      "static_refund_CallToSuicideTwice"
+    ],
+    "SystemOperationsTest" => [
+      "ABAcalls0",
+      "ABAcalls1",
+      "ABAcalls2",
+      "ABAcalls3",
+      "ABAcallsSuicide0",
+      "ABAcallsSuicide1",
+      "Call10",
+      "CallRecursiveBomb0",
+      "CallRecursiveBomb0_OOG_atMaxCallDepth",
+      "CallRecursiveBomb1",
+      "CallRecursiveBomb2",
+      "CallRecursiveBomb3",
+      "CallRecursiveBombLog",
+      "CallRecursiveBombLog2",
+      "CallToNameRegistrator0",
+      "CallToNameRegistratorAddressTooBigLeft",
+      "CallToNameRegistratorAddressTooBigRight",
+
```

Comment:
What's the significance of `{2,1}` and `{"passing", 0}` here?

Reply:
It's a [odd API](http://erlang.org/doc/man/ets.html#update_counter-4).

`{2,1}` is an `UpdateOp` which increments a counter by 1.

`{"passing", 0}` initializes the "passing" key to `0`

Reply:
Thanks for the explanation!